packages/runtime/src/state/modules/edit-mode.ts

```diff
@@ -0,0 +1,25 @@
+import { Action, ActionTypes } from '../actions'
+
+export const EditModes = {
+  BUILD: 'build',
+  CONTENT: 'content',
+  INTERACT: 'interact',
+} as const
```

Comment:
This is a minor nit, but I'm mentioning it because it used to be `BuilderEditMode` before and it was changed, but for enums we want to keep it singular, so `EditMode`. We want the type and the value to be the same so you don't have to import `EditModes` and `EditMode`

Reply:
Also, why did you drop `Builder` from the name? I'm trying to be as explicit as possible with naming in the runtime so that it's clear what's builder-specific vs what's part of the host